opentelemetry-api/src/opentelemetry/util/_time.py

```diff
@@ -0,0 +1,39 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from logging import getLogger
+from sys import version_info
+
+if version_info.minor < 7:
+    getLogger(__name__).warning(  # pylint: disable=logging-not-lazy
+        "You are using Python version 3.%s. This version does not include a "
```

Comment:
I think this is more verbose that it needs to be. It's a good code comment for maintainers, but for users it suffices to say "in this version of python, opentelemetry only supports millisecond-level granularity".

Reply:
In addition, why give the context that this private function will be removed once opentelemetry-api doesn't support python 3.7? I feel like that's somewhat obvious, and also unnecessary information to the user since 3.7+ supports NS granularity.

Reply:
Yes, you are right. I preferred to err on the safe side and be too verbose to make sure that our users *never* try using `_time_ns`. Refactoring this.